src/Components/Pages/SignUpPage.js

```diff
@@ -1,7 +1,6 @@
-import { Button, Card, Container, Form } from 'react-bootstrap';
+import { Card, Container, Button, Dropdown, DropdownButton, Form } from 'react-bootstrap';
```

Comment:
Make sure you don't have unused import statements.

Reply:
Removed unused libraries.